test/specs/apexReplayDebugger.e2e.ts

```diff
@@ -0,0 +1,173 @@
+
+    // Calling SFDX: Turn On Apex Debug Log for Replay Debugger fails on some machines.
+    // Reloading the window forces the extensions to be reloaded and this seems to fix
+    // the issue.
+    await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Reload Window', 10);
+    await utilities.pause(10);
+
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Turn On Apex Debug Log for Replay Debugger', 10);
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Turn On Apex Debug Log for Replay Debugger', fiveMinutes);
+
+    // Look for the success notification that appears which says, "SFDX: Turn On Apex Debug Log for Replay Debugger successfully ran".
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Turn On Apex Debug Log for Replay Debugger successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+  });
+
+  step('Run the Anonymous Apex Debugger with Currently Selected Text', async () => {
+    // Get open text editor
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+
+    // Open test file
+    const textEditor = await editorView.openEditor('ExampleApexClassTest.cls') as TextEditor;
+
+    // Select text
+    await textEditor.selectText('ExampleApexClass.SayHello(\'Cody\');');
+    await utilities.pause(1);
+
+    // Run SFDX: Launch Apex Replay Debugger with Currently Selected Text.
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Execute Anonymous Apex with Currently Selected Text', 1);
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running Execute Anonymous Apex', fiveMinutes);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'Execute Anonymous Apex successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+    await utilities.pause(1);
+  });
+
+  step('SFDX: Get Apex Debug Logs', async () => {
+    // Run SFDX: Get Apex Debug Logs
+    const workbench = await browser.getWorkbench();
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Get Apex Debug Logs', 1);
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Getting Apex debug logs', fiveMinutes);
+
+    // Select a log file
+    const quickPicks = await prompt.getQuickPicks();
+    expect(quickPicks).not.toBeUndefined();
+    expect(quickPicks.length).toBeGreaterThanOrEqual(1);
+    await prompt.selectQuickPick('User User - Api');
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Get Apex Debug Logs', fiveMinutes);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Get Apex Debug Logs successfully ran');
+    expect(successNotificationWasFound).toBe(true);
```

Comment:
the output panel should be cleared (line 87)
and then add validation of some of the contents in the .log file here,
and validation of some of the contents in the output panel